com.unity.render-pipelines.universal/Runtime/UniversalGlobalSettings.cs

```diff
@@ -0,0 +1,179 @@
+using UnityEditor;
+
+namespace UnityEngine.Rendering.Universal
+{
+    /// <summary>
+    /// Universal Render Pipeline's Global Settings.
+    /// Global settings are unique per Render Pipeline type. In URP, Global Settings contain:
+    /// - light layer names
+    /// </summary>
+    partial class UniversalGlobalSettings : RenderPipelineGlobalSettings
+    {
+        private static UniversalGlobalSettings cachedInstance = null;
+        public static UniversalGlobalSettings instance
+        {
+            get
+            {
+                if (cachedInstance == null)
+                    cachedInstance = GraphicsSettings.GetSettingsForRenderPipeline<UniversalRenderPipeline>() as UniversalGlobalSettings;
+                return cachedInstance;
+            }
+        }
+
+        static internal void UpdateGraphicsSettings(UniversalGlobalSettings newSettings)
+        {
+            if (newSettings == null || newSettings == cachedInstance)
+                return;
+            GraphicsSettings.RegisterRenderPipelineSettings<UniversalRenderPipeline>(newSettings as RenderPipelineGlobalSettings);
+            cachedInstance = newSettings;
+        }
+
+#if UNITY_EDITOR
+        //Making sure there is at least one UniversalGlobalSettings instance in the project
+        internal static UniversalGlobalSettings Ensure(string folderPath = "", bool canCreateNewAsset = true)
+        {
+            if (UniversalGlobalSettings.instance)
+                return UniversalGlobalSettings.instance;
+
+            UniversalGlobalSettings assetCreated = null;
+            string path = "Assets/" + folderPath + "/UniversalGlobalSettings.asset";
+            assetCreated = AssetDatabase.LoadAssetAtPath<UniversalGlobalSettings>(path);
+            if (assetCreated == null)
+            {
+                var guidHDGlobalAssets = AssetDatabase.FindAssets("t:UniversalGlobalSettings");
```

Comment:
```suggestion
                var guidURPGlobalAssets = AssetDatabase.FindAssets("t:UniversalGlobalSettings");
```

Nitpicking maybe a typo :P